app/helpers/measures_helper.rb

```diff
@@ -68,7 +68,9 @@ def control_line_wrapping_in_duty_expression(expression)
   def format_measure_condition_requirement(condition)
     case condition.measure_condition_class
     when 'threshold'
-      if condition.is_price_condition?
+      if condition.is_eps_condition_code_v?
```

Comment:
I think we could probably do with localising these strings? @jebw not sure how you feel

Reply:
I'd say not, we're not localising the rest of the site only RoO at the moment - if the decision is made to do that we need to do it en-masse, until then though it will just add another layer of indirection.